ui/settings_page.go

```diff
@@ -283,6 +280,23 @@ func (pg *settingsPage) subSectionSwitch(gtx layout.Context, title string, optio
 	})
 }
 
+func (pg *settingsPage) clickableRow(gtx layout.Context, title string, clickable *widget.Clickable, icon *widget.Icon, label decredmaterial.Label) layout.Dimensions {
+	return decredmaterial.Clickable(gtx, clickable, func(gtx C) D {
+		return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPaddingMinus5}.Layout(gtx, func(gtx C) D {
+			return pg.subSection(gtx, title, func(gtx C) D {
+				return layout.Flex{}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return label.Layout(gtx)
+					}),
+					layout.Rigid(func(gtx C) D {
+						return icon.Layout(gtx, values.MarginPadding22)
+					}),
+				)
+			})
+		})
+	})
+}
+
 func (pg *settingsPage) conditionalDisplay(gtx layout.Context, body layout.Widget) layout.Dimensions {
```

Comment:
Kindly add a boolean as one of the parameters of this method so it can accept the condition as an argument. With the "if" statement  inside the method.

Reply:
I'm not sure I fully get what you mean here

Reply:
`
func (pg *settingsPage) conditionalDisplay(gtx layout.Context, display, body layout.Widget) layout.Dimensions {
  if display {
    return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
		layout.Rigid(pg.lineSeparator()),
		layout.Rigid(body),
	)
 }
return layout.Dimensions{}
`